hooks/boot/01-kafo-hook-extensions.rb

```diff
@@ -37,7 +37,11 @@ def local_postgresql?
   end
 
   def local_redis?
-    (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || param_value('foreman_proxy::plugin::pulp', 'pulpcore_enabled')
+    (foreman_server? && !param_value('foreman', 'jobs_sidekiq_redis_url')) || param_value('foreman_proxy_plugin_pulp', 'pulpcore_enabled')
```

Comment:
The end of this line can be updated with the helper function `pulpcore_enabled?`